README.md

```diff
@@ -33,26 +33,64 @@ Next, configure the callbacks on the `ApplePaySessionPolyfill` object to return
 ApplePaySessionPolyfill.setMerchantIdentifier("My_Merchant_Identifier");
 
 // Optionally change the validation URL for merchant validation (the default is the URL for the Apple Pay Sandbox)
-//ApplePaySessionPolyfill.setValidationURL("https://someurl.somedomain.com")
+// ApplePaySessionPolyfill.setValidationURL("https://someurl.somedomain.com")
 
 // Re-declare the function to create a PaymentContact for billing
+// Check https://developer.apple.com/documentation/apple_pay_on_the_web/applepaypayment/1916097-shippingcontact
+// "phoneticGivenName" and "phoneticFamilyName" for API Version >= 3
 ApplePaySessionPolyfill.createBillingContact = function (session) {
   return {
-    /* ... */
+    givenName: 'Ravi',
+    familyName: 'Patel',
+    phoneticGivenName: 'Ravi',
+    phoneticFamilyName: 'Patel',
+    emailAddress: 'ravipatel@example.com',
+    phoneNumber: '(408) 555-5555',
```

Comment:
Reordered to match the declaration [here](https://developer.apple.com/documentation/apple_pay_on_the_web/applepaypaymentcontact#declaration):

```suggestion
    phoneNumber: '(408) 555-5555',
    emailAddress: 'ravipatel@example.com',
    givenName: 'Ravi',
    familyName: 'Patel',
    phoneticGivenName: 'Ravi',
    phoneticFamilyName: 'Patel',
```